sparkmagic/sparkmagic/kernels/kernelmagics.py

```diff
@@ -267,6 +268,32 @@ def spark(self, line, cell="", local_ns=None):
         else:
             return
 
+    @magic_arguments()
```

Comment:
I based this off of what the `%%spark` handler was doing so it should take all the same args as the `%%spark` handler so it could run `execute_spark`

It doesn't actually need any arguments, so I'll remove this